main.go

```diff
@@ -34,7 +35,50 @@ func initBucket() {
 	emptyVersionsFile, _ := json.Marshal(versions)
 	fmt.Println("Uploading empty " + versionFileKey + " file")
 
-	autoupdate.UploadS3File(s3Bucket, versionFileKey, bytes.NewReader(emptyVersionsFile))
+	err := autoupdate.UploadS3File(s3Bucket, versionFileKey, bytes.NewReader(emptyVersionsFile))
+	if err != nil {
+		panic(err)
+	}
+}
+func downloadLatest() {
+	appName := flag.Arg(1)
+	channel := flag.Arg(2)
+	s3Bucket := flag.Arg(3)
+	outputName := flag.Arg(4)
+	if flag.NArg() != 5 {
+		printUsage()
+		os.Exit(1)
+	}
+	versionFileKey := autoupdate.GetVersionFileKey(appName, channel)
+	fmt.Println("Fetching " + versionFileKey + "file")
+
+	versionFile, err := autoupdate.GetS3File(s3Bucket, versionFileKey, false, nil)
+
+	if err != nil {
+		panic(err)
+	}
+
+	var versions autoupdate.VersionFile
+
+	err = json.Unmarshal(versionFile, &versions)
+	if err != nil {
+		panic(err)
+	}
+
+	releaseFileKey := autoupdate.GetFileKey(appName, channel, versions.LastVersion+"/"+"windows-amd64")
+	fmt.Println("Fetching " + releaseFileKey + "file")
```

Comment:
@MichalPodeszwa see above as well, do we want to have an extra space after the `"` in `"file"` to make it read better when viewing the log?